lib/src/migrators/division.dart

```diff
@@ -36,9 +36,13 @@ More info: https://sass-lang.com/d/slash-div""";
   bool get isPessimistic => argResults['pessimistic'] as bool;
 
   @override
-  Map<Uri, String> migrateFile(Uri entrypoint) =>
-      _DivisionMigrationVisitor(this.isPessimistic, migrateDependencies)
-          .run(entrypoint);
+  Map<Uri, String> migrateFile(Uri entrypoint) {
+    var visitor =
+        _DivisionMigrationVisitor(this.isPessimistic, migrateDependencies);
+    var result = visitor.run(entrypoint);
+    missingDependencies.addAll(visitor.missingDependencies);
```

Comment:
Bleh, it sucks that each subclass has to do this manually rather than just having "warn for missing dependencies" be the automatic default behavior. It's not something that needs to happen in this PR, but it's worth thinking about how you could make this more automatic.

Reply:
Yeah, that's why I originally passed in the list for the visitor to add to, but this is probably cleaner than that, particularly right now when there's only one migrator that actually uses `missingDependencies`. Created #64 for this.